regulations/tasks.py

```diff
@@ -120,3 +133,13 @@ def make_s3_client():
         aws_secret_access_key=settings.ATTACHMENT_SECRET_ACCESS_KEY,
     )
     return session.client('s3')
+
+
+def validate_attachments(files):
+    if len(files) > MAX_ATTACHMENT_COUNT:
+        return False, "Too many attachments"
+    for file_ in files:
```

Comment:
Could we handle bad file types in `regulations/views/comment:upload_proxy` instead? We can reject requests that want to upload prohibited file types there--we're already rejecting requests to upload files that are too big there.